src/components/issue-list.jsx

```diff
@@ -42,21 +42,23 @@ function toCSVString(table) {
 }
 
 
-const IssueList = React.createClass({
-  displayName: 'IssueList',
-  getInitialState() {
-    return {morePressedCount: 0, showCSVModal: false};
-  },
-  showAllIssues() {
+class IssueList extends Component {
+  static displayName = 'IssueList';
```

Comment:
Not necessary if it's the same as class or function name.
There's a few, if not all of `displayName` in this patch is not needed anymore.

https://reactjs.org/docs/react-component.html#displayname